lib/Froxlor/Validate/Validate.php

```diff
@@ -64,6 +64,26 @@ public static function validate($str, $fieldname, $pattern = '', $lng = '', $emp
 		exit();
 	}
 
+    /**
+     * Converts CIDR to a netmask address
+     *
+     * @thx to https://stackoverflow.com/a/5711080/3020926
+     * @param string $cidr
+     *
+     * @return string
+     */
+    public static function cidr2NetmaskAddr ($cidr) {
+
+        $ta = substr ($cidr, strpos ($cidr, '/') + 1) * 1;
+        $netmask = str_split (str_pad (str_pad ('', $ta, '1'), 32, '0'), 8);
+
+        foreach ($netmask as &$element) {
+            $element = bindec ($element);
+        }
+
+        return join ('.', $netmask);
```

Comment:
`join` is just an alias for `implode`, please use `implode` because 

- we always use `implode`
- you never know when an alias might get removed from php in later versions

Reply:
Changed.